pkg/init/Dockerfile

```diff
@@ -1,9 +1,20 @@
-FROM alpine:3.5
+FROM linuxkit/alpine:f0169b60fb260d74025496ae6fd93213fecaba8f@sha256:23743c7206ebe8a609442c5ac7084a26ed45ce8f5213960428bca264225849f1 AS build
+# XXX There is an issue with the bsd-compat-headers package I don't
+# understand. Installing the file directly is yuk but works
```

Comment:
I build the new alpine base locally with `make tag` and then used the new image:
```FROM linuxkit/alpine:a09caa34e52e873b23114cd1d691a9a61ee1a10f AS build
RUN apk add \
    bsd-compat-headers \
    gcc \
    linux-headers \
    musl-dev
COPY ./start-stop-daemon.c .
```
and still get the error:
```
[no-c *]rn-mbp:t/linuxkit/pkg/init: make tag
docker build --no-cache --network=none -t linuxkit/init:4c7e24873e2239c04ce2cca2fb4ddfcc173be459 .
Sending build context to Docker daemon  43.01kB
Step 1/10 : FROM linuxkit/alpine:a09caa34e52e873b23114cd1d691a9a61ee1a10f AS build
 ---> ee89320575c0
Step 2/10 : RUN apk add     bsd-compat-headers     gcc     linux-headers     musl-dev
 ---> Running in 53659afe41e3
ERROR: bsd-compat-headers-0.7.1-r0: package mentioned in index not found (try 'apk update')
(1/15) Installing bsd-compat-headers (0.7.1-r0)
(2/15) Installing binutils-libs (2.28-r1)
(3/15) Installing binutils (2.28-r1)
(4/15) Installing gmp (6.1.2-r0)
(5/15) Installing isl (0.17.1-r0)
(6/15) Installing libgomp (6.3.0-r4)
(7/15) Installing libatomic (6.3.0-r4)
(8/15) Installing pkgconf (1.3.5-r0)
(9/15) Installing libgcc (6.3.0-r4)
(10/15) Installing mpfr3 (3.1.5-r0)
(11/15) Installing mpc1 (1.0.3-r0)
(12/15) Installing libstdc++ (6.3.0-r4)
(13/15) Installing gcc (6.3.0-r4)
(14/15) Installing linux-headers (4.4.6-r2)
(15/15) Installing musl-dev (1.1.16-r8)
Executing busybox-1.26.2-r2.trigger
1 errors; 103 MiB in 25 packages
```